src/riak_core_app.erl

```diff
@@ -145,3 +146,12 @@ register_capabilities() ->
       end,
       Capabilities),
     ok.
+
+%% TODO: This belongs in riak_kv - an issue will be created to move it, but time constraints.
```

Comment:
Because `riak_core` uses `eleveldb` for its hashtree implementation, it's always available - the app itself doesn't have initialization of its own (especially now that we've moved this over to `riak_core`.